enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/messaging/CoreReplicatedContentMarshal.java

```diff
@@ -85,6 +87,11 @@ else if ( content instanceof DistributedOperation )
             channel.put( DISTRIBUTED_OPERATION );
             ((DistributedOperation) content).serialize( channel );
         }
+        else if ( content instanceof DummyRequest )
```

Comment:
Is there a reason we don't want to call a method on `ReplicatedContent`? If so could we use a visitor pattern instead?